.env.example

```diff
@@ -1 +1 @@
-OPENAI_API_KEY=
\ No newline at end of file
+OPENAI_API_KEY=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
```

Comment:
Please never publish your API key in your repository code. You should only include it in files that are gitignored.